### PR TITLE
updated shutdown text

### PR DESCRIPTION
**Pull request by femans**

### CODE REVIEW

File: src/components/HomeView.vue [hunk 1]
```diff
@@ -28,15 +28,14 @@ onBeforeMount(() => {
...
+            ecosystem amidst operational costs and our personal sustenance needs. 
           </p>
 
           <p>
-            We stand at a critical crossroads, committed to maintaining our crucial role in the Web3
-            ecosystem amidst operational costs and our personal sustenance needs. Unless additional
-            funding comes through, we will need to shut down our public APIs in the coming weeks. As
-            of now, site search is already suspended, but API access remains.
+            Unless additional funding comes through,
+            <b>we will to shut down our public APIs June the 7th, 2023</b>, we don't know for how
```
Comment (dokterbob) [inline src/components/HomeView.vue]: will have to shut down

**Review by dokterbob — CHANGES_REQUESTED** [review]

File: src/components/HomeView.vue (hunk 1 above)
Comment (femans) [reply src/components/HomeView.vue]: will shut down

File: src/components/HomeView.vue [hunk 2]
```diff
@@ -29,13 +29,13 @@ onBeforeMount(() => {
...
             We stand at a critical crossroads, committed to maintaining our crucial role in the Web3
-            ecosystem amidst operational costs and our personal sustenance needs. 
+            ecosystem amidst operational costs and our personal sustenance needs.
           </p>
 
           <p>
             Unless additional funding comes through,
-            <b>we will to shut down our public APIs June the 7th, 2023</b>, we don't know for how
-            long. As of now, site search is already suspended, but API access remains until then.
+            <b>we will shut down our public APIs June the 7th, 2023</b>, we don't know for how long.
```
Comment (dokterbob) [inline src/components/HomeView.vue:37]: I would suggest the word suspend, skipping the 'we don't know for how long'. But not essential.

Otherwise o.k.

**Review by dokterbob — APPROVED** [review]

File: src/components/HomeView.vue (hunk 2 above)
Comment (femans) [reply src/components/HomeView.vue:37]: signal that we may come back, depending on evolving circumstances. Otherwise it seems definitive.

**Pull request merged**